src/analysis/analytic/composition.lean

```diff
@@ -838,10 +836,7 @@ begin
   rcases j with ⟨a', b'⟩,
   rintros ⟨h, h'⟩,
   have H : a = a', by { ext1, exact h },
-  induction H,
-  congr,
-  ext1,
-  exact h'
+  induction H, congr, ext1, exact h'
```

Comment:
```suggestion
  induction H, congr' with n : 1, exact h'
```
?

Reply:
there is no variable introduced by this `ext1`. I think this looks better than `congr' with : 1`